resources/views/device-group/index.blade.php

```diff
@@ -57,6 +57,39 @@ class="fa fa-trash" aria-hidden="true"></i></button>
             </div>
         </div>
     </div>
+    <div class="container-fluid">
+        <div id="manage-device-groups-panel" class="panel panel-default">
```

Comment:
This id is duplicated, changed it please.

Reply:
changed